src/integTest/resources/android-multiple-flavors/build.gradle

```diff
@@ -0,0 +1,53 @@
+buildscript {
+  repositories {
+    google()
+    mavenCentral()
+    jcenter()
+  }
+
+  // Mapping between Gradle and Android plugin versions: https://developer.android.com/studio/releases/gradle-plugin
+  def androidGradlePluginVersion = '3.0.0'
+  if (gradle.gradleVersion.compareTo('6.1.1') >= 0) {
+    androidGradlePluginVersion = '4.0.0'
+  } else if (gradle.gradleVersion.compareTo('5.6.4') >= 0) {
+    androidGradlePluginVersion = '3.6.4'
+  } else if (gradle.gradleVersion.compareTo('5.0') >= 0) {
+    androidGradlePluginVersion = '3.3.3'
+  }
+  dependencies {
```

Comment:
minor. could do with a new line before this block.

Reply:
> minor. could do with a new line before dependencies section.

Done: 49f3eb6e60a343ed61f8f69bc2f937f68da4995d